lxd/db/instances_test.go

```diff
@@ -202,7 +216,13 @@ func TestInstanceList(t *testing.T) {
 	var instances []db.Instance
 	err = cluster.InstanceList(nil, func(dbInst db.Instance, p db.Project, profiles []api.Profile) error {
 		dbInst.Config = db.ExpandInstanceConfig(dbInst.Config, profiles)
-		dbInst.Devices = db.ExpandInstanceDevices(deviceConfig.NewDevices(dbInst.Devices), profiles).CloneNative()
+		for _, p := range profiles {
```

Comment:
@masnax why isnt this using `db.ExpandInstanceDevices` anymore?

Reply:
@masnax

Reply:
Answered in a latter comment.